README.md

```diff
@@ -6,13 +6,19 @@ This repository includes a library of mature and curated [Bicep][Bicep] modules
 
 The CI environment supports both ARM and Bicep and can be leveraged using GitHub actions as well as Azure DevOps pipelines.
 
-<h1 style="color: steelblue;">Upcoming breaking changes</h1>
+<h1 style="color: steelblue;">⚠️ Upcoming breaking changes ⚠️</h1> 
 
-In between now and the release of version `0.11.0`, the `main` branch is subject to several upcoming breaking changes that will affect all modules (e.g., the renaming of folders and files).
+Following the recent release of [`0.11.0`](https://github.com/Azure/ResourceModules/releases/tag/v0.11.0), the upcoming period will focus on implementing the remaining changes required to align the modules to the developed [Azure Verified Modules](aka.ms/avm) specifications. This will enable us to move & publish the library to the official [Public Bicep Registry](https://github.com/Azure/bicep-registry-modules).
 
-The rationale is an ongoing effort to prepare our modules for a release in the official [Public Bicep Registry](https://github.com/Azure/bicep-registry-modules), forcing us to align the structural requirements.
+Please note that these changes will affect many interfaces (e.g., the diagnostic settings). We intend to keep this period as small as possible, but are limited by our own available capacity. As we want to avoid one 'big bang' migration, we will incrementally align & move modules, but keep a copy in this repository until the move is concluded. For modules that were already published, we will redirect the proposed changes to the new [repository](https://github.com/Azure/bicep-registry-modules) and it's AVM folder. In its final state, this folder will continue all modules you can currently find in this repository's 'modules' folder.
```

Comment:
```suggestion
Please note that these changes will affect many interfaces (e.g., the diagnostic settings). We intend to keep this period as small as possible, but are limited by our own available capacity. As we want to avoid one 'big bang' migration, we will incrementally align & move modules, but keep a copy in this repository until the move is concluded. For modules that were already published, we will redirect the proposed changes to the new [repository](https://github.com/Azure/bicep-registry-modules) and its AVM folder. In its final state, this folder will continue all modules you can currently find in this repository's 'modules' folder.
```